docs/userGuide/syntax/questions.mbdf

```diff
@@ -1,130 +1,325 @@
-## Questions
+## Questions and Quizzes
 
-**Question component consists of a question body, a hint and an answer.**
+Question and quiz components provide an easy way to test readers on the relevant content topic in the page.
 
-<include src="codeAndOutput.md" boilerplate >
-<variable name="highlightStyle">html</variable>
-<variable name="code">
-<question>
-  The question body. <md>:grey_question:</md>
-  <div slot="hint">
-    Question hint. <md>:crystal_ball:</md>
-  </div>
-  <div slot="answer">
-    Question answer. <md>:pencil:</md>
-  </div>
+#### Introduction
+
+Question components (`<question>`) can be one of the following types: **Mcq**, **Checkbox** or **Text**.
+
+<div class="px-5">
+<question type="checkbox" header="Which of the following is true? (**Card Header**)" hint="This is the question hint, you can use **markdown** here as well.">
```

Comment:
`Card` is a bootstrap term, not used in the MarkBind vocabulary (from author POV)?